config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder.yaml

```diff
@@ -0,0 +1,64 @@
+# periodic jobs
+
```

Comment:
the CI is failing possibly because the file needs `peridics:` on top:

> 2019/06/05 21:09:04 Failed to read config file: error unmarshaling config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder.yaml: error unmarshaling JSON: while decoding JSON: 

see https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml#L3

Reply:
Thanks, typo on my side.